### ISSUE TITLE
Install `linter` automatically.

### ISSUE THREAD

**Issue by kepler0** [hebaishi/linter-gcc#6]
By using [`atom-package-deps`](https://github.com/steelbrain/package-deps) you are able to install `linter` automatically on activation. This will allow new Atom users to install only your package and nothing else, removing one step from the installation process.

Please see AtomLinter/linter-ruby#38 for example.

**Comment by hebaishi** [comment]
Quick question - does this mean the user has to manually install ```atom-package-deps``` for this to work?

**Comment by k2b6s9j** [comment]
Nope!

**Comment by hebaishi** [comment]
Sounds good! I'll try to make the changes at some point this week.

**Comment by hebaishi** [comment]
Added
```js
require("atom-package-deps").install("linter-gcc");
```
to my activate function and it just gave an error saying atom-package-deps could not be found. I thought atom-package-deps was a core package. I've removed it for now.

**Comment by kepler0** [comment]
`atom-package-deps` isn't a core package. It's a Node Package Manager module, created by @steelbrain, much like the `atom-linter` module, to install a list of package dependencies.

**Comment by hebaishi** [comment]
Gotcha... I've added ```atom-package-deps``` to the dependences list, and put ```linter``` in the ```package-deps``` but it still won't linstall linter. Am I doing something wrong?

**Comment by steelbrain** [comment]
@hebaishi Just a side note that it doesn't install anything if it's already installed

**Comment by hebaishi** [comment]
Got it working! Small error in the package manifest. Working brilliantly now! Thanks a lot @steelbrain for the amazing work! I love the progress bar!
